build.gradle

```diff
@@ -45,6 +45,9 @@ allprojects {
         maven {
             url "https://oss.sonatype.org/content/repositories/snapshots"
         }
+        maven {
+            url "https://dl.bintray.com/yoyama/maven"
+        }
```

Comment:
so you stop using maven central?:-)

Reply:
Yes. Sonatype is too slow and frequently errors :(